Automattic/WP-Job-Manager
# Add sections active and inactive on the WPJM Licenses page

Fixes #2396
Based on #2392 

### Changes proposed in this Pull Request

* Add sections "active" and "inactive" for the WPJM Licenses page;

### Testing instructions

1. Verify if the list of addons for WPJM is grouped by active and inactive;
2. Verify if the buttons to activate a license and deactivate a licence still work, and that the product is moved to the correct place;

### Screenshots

![image](https://user-images.githubusercontent.com/529864/231766468-8b122edc-c7f4-4e77-be67-ebb49de127d7.png)
